### DISCUSSION TITLE
Get the number of existing registry subitems

### DISCUSSION THREAD

**Discussion by cz-dev-ge** [wixtoolset/issues#7428] [Questions] [answered]
Hi there,
is it possible to find out if multiple items matching a pattern exist in a specific registry path?

The reason is, RabbitMQ states that only one version of Erlang must be installed. Since I want to bundle it I must make sure this is true. I want to use registry search to find out how many items in the registry match this pattern:

`HKLM:\SOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Uninstall\Erlang*`

If it is more than one I want to fail the installation.

Best regards.

**Comment by robmen** [comment, marked as answer]
That sort of search functionality is not built into the Windows Installer. You would need a custom action in this case.

**Reply by cz-dev-ge** [reply]
Thanks.
